### PR TITLE
Update coq-coqffi.dev, add coq-coqffi.1.0.0~beta8

### PR DESCRIPTION
**Pull request by lthms**
Long time no see, dear maintainers. (:

### CODE REVIEW

**Comment by lthms** [comment]
Hum, it looks like the CI is failing because `coqffi` does not support (yet) recent enough version of Coq.

I can imagine why it could be an issue, but in the same time, I am lacking time to support newer version of Coq, while I would benefit from having these changes merged.

Any idea of how to proceed?

File: extra-dev/packages/coq-coqffi/coq-coqffi.dev/opam [hunk 1]
```diff
@@ -17,10 +17,10 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
-  "ocaml" {>= "4.08" & < "4.12~" }
+  "ocaml" {>= "4.08" & < "5.0~" }
```
Comment (palmskog) [inline extra-dev/packages/coq-coqffi/coq-coqffi.dev/opam:20]: @lthms if your OCaml code is fundamentally incompatible with OCaml 5, the preferred way to state this in opam packages is as follows:
```
conflicts: [
  "base-nnp"
  "ocaml-option-nnpchecker"
]
```

Comment (lthms) [reply extra-dev/packages/coq-coqffi/coq-coqffi.dev/opam:20]: I don’t think it is fundamentally incompatible, I just didn’t try/adapt it yet.

**Comment by lthms** [comment]
Thanks @palmskog!

**Pull request merged**